hardware/main.py

```diff
@@ -0,0 +1,22 @@
+import time
+from CommunicationsPi.lan_client import LANClient
+from SensorPi.sense_pi import SensePi
+
+# This file shows the usage of SensePi and LANClient
+sensePi = SensePi()
+client = LANClient()
+
+while True:
+    print("iteration")
```

Comment:
Considering Jack's comment on auxiliary outputs from the hardware tests, we should take as less terminal lines as possible unless there is something broken. Can we log this instead? or is this indeed required.

Reply:
This is just a demo script. We need more stories to improve this. Definitely we should use print as less as possible. Will do at later phase.

Reply:
Got it.